### ISSUE TITLE
sql: consider printing always-on trace in slow query log

### ISSUE THREAD

**Issue by tbg** [cockroachdb/cockroach#55591]
Once we have always-on tracing, the slow query log could print the trace corresponding to the request, and in turn hopefully explain why the request is slow.

**Comment by tbg** [comment]
In light of https://github.com/cockroachdb/cockroach/issues/55589#issuecomment-709949302, it doesn't seem worth changing anything about the slow query log.

**Comment by knz** [comment]
yes there are non-debugging reasons. However I don't recall the detail. This is more a question for @jordanlewis
